.github/workflows/ci-cd.yml

```diff
@@ -369,6 +369,8 @@ jobs:
       - run: yarn cypress install
       - name: cypress
         env:
+          CYPRESS_RAILS_HOST: localhost
+          CYPRESS_RAILS_PORT: 3000
```

Comment:
@maestromac Pinned the Cypress domain on CI as well to match the `APP_DOMAIN` env variable - otherwise, navigating using dropdowns (which contain full permalinks) in the end-to-end test I added doesn't work